src/main/java/com/conveyal/r5/analyst/cluster/AnalystWorker.java

```diff
@@ -421,6 +421,7 @@ private void handleStaticSiteRequest (StaticSiteRequest.PointRequest request, Tr
 
         if (request.request.bucket != null) computer.run();
         else {
+            lastHighPriorityRequestProcessed = System.currentTimeMillis();
```

Comment:
Might be worth adding a comment about why it's only a highPriorityRequest when bucket != null (because batch regional analyses save results to buckets, while one-off analyses don't)